lib/i18n/american-english.js

```diff
@@ -300,6 +300,22 @@ class EnglishLanguagePack extends DefaultLanguagePack {
         return [inflected, ...words.slice(1)].join(' ');
     }
 
+    toAgentSideUtterance(phrase) {
+        return phrase.replace(/\b(i|me|my|mine)\b/g, (what) => {
+            switch(what) {
+            case 'me':
+            case 'i':
+                return 'you';
+            case 'my':
+                return 'your';
+            case 'mine':
+                return 'yours';
+            default:
+                return what;
+            }
+        });
+    }
+
```

Comment:
Why is this not turning "Alright, I shuffled your Spotify." to "Alright, you shuffled your Spotify."

Reply:
Because we only apply it to the canonical forms in Thingpedia. The words in the templates are unchanged.
This is also why I need to do it this way instead of adding to the postprocessing in the sentence.